tests/manage/test_create_pod_attaching_multiple_PVC_start_IO.py

```diff
@@ -0,0 +1,180 @@
+from tests import helpers
+from ocs_ci.framework import config
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope='class')
+def ocs260_fixture(request):
+
+    self = request.node.cls
+
+    def finalizer():
+        teardown(self)
+    request.addfinalizer(finalizer)
+    setup()
+
+
+def setup():
+    """
+    Setting up the environment for the test
+    """
+    log.info("Create Ceph Block Pool")
+    global RBD_POOL
+    RBD_POOL = helpers.create_ceph_block_pool(pool_name="cirrostest")
+
+    log.info("Create RBD Secret")
+    global RBD_POOL_SECRET_OBJ
+    RBD_POOL_SECRET_OBJ = helpers.create_secret(constants.CEPHBLOCKPOOL)
+
+    log.info("Create RBD Storage Class")
+    global RBD_STORAGE_CLASS_OBJ
+    RBD_STORAGE_CLASS_OBJ = helpers.create_storage_class(
+        constants.CEPHBLOCKPOOL,
+        interface_name=RBD_POOL.name,
+        secret_name=RBD_POOL_SECRET_OBJ.name
+    )
+
+
+def teardown(self):
+    """
+    Tearing down the environment
+    """
+    dc = ocp.OCP(
+        kind=constants.DEPLOYMENTCONFIG, namespace=config.ENV_DATA['cluster_namespace']
+    )
+    dc.delete(resource_name=self.cirros_name[0], wait=True)
+    pvc.delete_all_pvcs()
+    RBD_STORAGE_CLASS_OBJ.delete()
+    RBD_POOL_SECRET_OBJ.delete()
+    RBD_POOL.delete()
+
+
+def create_pvc(pvc_name_prefix, pvc_count):
+    """
+    Function to create multiple PVC based on count and name
+
+    Args:
+        pvc_name_prefix (str): User defined pvc prefix name
+        pvc_count (int): Specify number of pvc count to be create
+
+    Returns:
+        pvc_list (list): Returns the pvc created
+    """
+    pvc_list = []
+    for count in range(pvc_count):
+        name = pvc_name_prefix + str(count)
+        pvc_obj = helpers.create_pvc(sc_name=RBD_STORAGE_CLASS_OBJ.name, pvc_name=name)
+        pvc_list.append(pvc_obj.name)
+    return pvc_list
+
+
+def create_pod(pvc_list):
+    """
+    Function to create Cirros pod with list of pvc.
+
+    Args:
+        pvc_list (list): List of PVC to be mounted in the cirros pod
+
+    Returns:
+        cirros_pod_name (str): Cirros pod deployment config name
+    """
+    temp_dict1 = []
+    temp_dict2 = []
+    temp_str = ''
+    cirros_yaml_data = templating.load_yaml_to_dict(constants.CIRROS_APP_POD_YAML)
+    # Following code collects the existing cirros yaml data, removes some
+    # values which will be updated as per pvc_list argument
+    cirros_pod_name = cirros_yaml_data['metadata']['name']
+    cirros_yaml_data['metadata']['namespace'] = config.ENV_DATA['cluster_namespace']
+    temp_list1 = cirros_yaml_data['spec']['template']['spec']['containers'][0]['args']
+    temp_list1.pop(len(temp_list1) - 1)
+    temp_list2 = cirros_yaml_data['spec']['template']['spec']['containers'][0][
+        'livenessProbe'
+    ]['exec']['command']
+    temp_list2.pop(len(temp_list2) - 1)
+    # for loop which creates the values, which will be added in cirros.yaml
+    for pv in pvc_list:
+        temp_dict1.append(
```

Comment:
please create a function to 'attach_pvc_to_pod()' under pod.py or even better, 'attch_pvc()' directly under Pod object in pod.py